src/Infiltration/ui/Victory.tsx

```diff
@@ -79,9 +81,11 @@ export function Victory(props: IProps): React.ReactElement {
       <Box sx={{ width: "fit-content" }}>
         <Box sx={{ width: "100%" }}>
           <Select value={factionName} onChange={changeDropdown} sx={{ mr: 1 }}>
-            <MenuItem key={"none"} value={"none"}>
-              {"none"}
-            </MenuItem>
+            {defaultFaction === "none" && (
```

Comment:
I think we should keep the "none" option unconditionally, just for consistency/maybe someone wants to throw away rep for some mad reason

Reply:
I disagree on the need for consistency (why include a none option when it isn't necessary) but I can understand that argument, however I'm not sure why we would enable throwing away rep even if people want to. If they don't care where it goes they can just let it go to the default faction, which is most likely the best one to pick.

Reply:
Ok, the button doesn't actually function when none is selected, so it's not like a usable (even if silly) option is being removed.

Just waiting on testing